### ISSUE TITLE
Sprite, load from url doesn’t seem to work

### ISSUE THREAD

**Issue by zumwaltwood** [Scirra/Construct-3-bugs#1593]
## Problem description

*Enter a concise description of your problem here*
Simply trying to get an image to pull from a url that is a gif
[testloadurl.zip](https://github.com/Scirra/Construct-3-bugs/files/2010829/testloadurl.zip)

## Attach a .c3p

*A minimal Construct 3 project (.c3p) is required to be attached. Please see the guidelines*

## Steps to reproduce

1. Create a sprite object
2. Add to layout
3. Add a system event on layout start, action to load a sprite from url target is the sprite you created

## Observed result

*What do you see happen?*
Nothing

## Expected result

*What did you expect to happen instead?*
Load the image

## Affected browsers

* Safari: yes

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

Platform information
Browser: Safari
Browser version: 11.0
Browser engine: WebKit
Browser architecture: (unknown)
Context: browser
Operating system: iOS
Operating system version: 11.3
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 2
Logical CPU cores: (unavailable, defaulting to 2)
Approx. device memory: (unavailable)
User agent: Mozilla/5.0 (iPad; CPU OS 11_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.0 Mobile/15E148 Safari/604.1
C3 release: r100 (beta)
Language setting: en-US

Local storage
Storage quota (approx): (status unavailable)
Storage usage (approx): (status unavailable)
Persistant storage: (status unavailable)

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Rendering multiple on-screen Layout Views is slow in Safari due to bug 177132
CSS containment is not supported. Editor performance may be significantly degraded.
The <dialog> element is not supported. A polyfill is in use.
The Clipboard API is not supported. Some clipboard features may be unavailable.
Web Animations are not supported. Animations are disabled.
This browser does not support pluralisation rules. Plural forms may not be correct.
WebGL 2+ is not supported. Rendering quality and features may be affected.
ImageBitmap is not supported. Texture loading performance may be degraded.
Idle callbacks are not supported. Background loading performance may be degraded.
Downloading local content is not supported.
Determining input device capabilities is not supported.
Persistent storage is not available. Local storage could be deleted by the browser.
Storage quota estimate is unavailable.
WebGL information
Version string: WebGL 1.0 (OpenGL ES 2.0 Metal - 52.3.2)
Numeric version: 1
Supports NPOT textures: partial
Supports GPU profiling: no
Vendor: Apple Inc.
Renderer: Apple A10X GPU
Major performance caveat: no
Maximum texture size: 4096
Point size range: 1 to 511
Extensions:

EXT_blend_minmax
EXT_sRGB
OES_texture_float
OES_texture_float_linear
OES_texture_half_float
OES_texture_half_float_linear
OES_standard_derivatives
EXT_shader_texture_lod
EXT_texture_filter_anisotropic
OES_vertex_array_object
OES_element_index_uint
WEBGL_lose_context
WEBKIT_WEBGL_compressed_texture_pvrtc
WEBGL_depth_texture
ANGLE_instanced_arrays
WEBGL_debug_renderer_info
Audio information
System sample rate: 44100 Hz
Output channels: 0
Output interpretation: speakers
Supported decode formats:

MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats: (encoding not supported)

Video information
Supported decode formats:

H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats: (encoding not supported)

</details>

**Comment by AshleyScirra** [comment]
It's blocked by the browser for security reasons - see the error in the browser console:

> Access to Image at 'https://secure.trust-guard.com/seals/privacy/13659-mini.gif' from origin 'https://preview.construct.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://preview.construct.net' is therefore not allowed access.

See the section on "Cross-domain requests" here for more info: https://www.construct.net/tutorials/publishing-to-the-web-10

**Comment by zumwaltwood** [comment]
So it sounds like c3 needs to send cor info

**Comment by zumwaltwood** [comment]
<httpProtocol>
		<customHeaders>
			<add name="Access-Control-Allow-Origin" value="*" />
		</customHeaders>
	</httpProtocol>

i have this in my web.config for cross domain

**Comment by shortercode** [comment]
If you check the response headers from your server you will see it is not adding the "Access-Control-Allow-Origin" header. I'm not sure what your using for your server, but if your sure your config is correct try restarting the server I guess.

**Comment by zumwaltwood** [comment]
iis 7 on windows server 2012 R2, going to try the .htaccess and include IfModule settings and see if that helps, I see that error in debug console, I also see load errors for image files and sound files, I’ll bug report that later but you can see the same 403 errors if you look at debugger from my cyborgsattack.com

**Comment by zumwaltwood** [comment]
Unfortunately the image changes daily so I need to download the updated image daily then post it out, ran into a new issue though, I tried to force only requests from cyborgsattack.com and the error came back that the origin of cyborgsattack.com didn't match the requested origin of cyborgsattack.com so it refused the request, so I had to use wildcard for now until I have time later this weekend to hammer on it.
